cleanlab/internal/token_classification_utils.py

```diff
@@ -29,42 +30,38 @@ def get_sentence(words: list) -> str:
 
 
 def filter_sentence(
-    sentences: list, condition: Optional[Callable] = None, return_mask: bool = True
-) -> Union[Tuple[list, list], list]:
+    sentences: List[str],
+    condition: Optional[Callable[[str], bool]] = None,
+) -> Tuple[List[str], List[bool]]:
     """
     Filter sentence based on some condition, and returns filter mask
 
     Parameters
     ----------
-        sentences: list
+        sentences: List[str]
             list of sentences
 
-        condition: Callable
+        condition: Optional[Callable[[str], bool]]
             sentence filtering condition
 
-        return_mask: bool
-            if set to True, also returns mask
-
     Returns
     ---------
-        sentences: list
+        sentences: List[str]
             list of sentences filtered
 
-        mask: if `return_mask`, also returns a mask such that `mask[i] == True` if the i'th sentence is included in the
-        filtered sentence, otherwise `mask[i] == False`
+        mask: List[bool]
+            boolean mask such that `mask[i] == True` if the i'th sentence is included in the
+            filtered sentence, otherwise `mask[i] == False`
```

Comment:
Save for future PR where you clean up the docstrings and their formatting (eg. capitalization), but each of these docstrings should have some example of what could possibly be passed in here for this argument, otherwise it remains too vague for users.